test cases/wasm/3 jslib/somefuncs.js

```diff
@@ -0,0 +1,3 @@
+function sampleFunction() {
+    alert("Something happened!");
+}
```

Comment:
```suggestion
mergeInto(LibraryManager.library, {
    sample_function__sig: 'v',
    sample_function: function() {
        alert("Something happened!");
    },
});
```

Reply:
note the `__sig` is important for emscripten (custom) dynamic linking to work (since it provides the signature of the function to be exposed to side modules).

Reply:
Fixed, thanks.